data/korath/korath missions.txt

```diff
@@ -257,3 +257,548 @@ mission "Small Scale Delivery"
+			`	Pleased with this success, Akafavah continues, "Korath," gesturing at the two pilots.`
+			`	"I'm a human," you say, "from Earth."`
+			`	Akafavah says suddenly, "Oi! Humanika!" It's unclear what emotion the Korath is expressing, but it is a strong one.`
+			`	They see your confusion and seem to give up on communicating with words. It's time for a different approach.`
+				goto map
+
+			label attack
+			`	You reach for your gun. They move faster than you would have expected possible. One of strikes a pose with their arms and shouts, "Nag! Nag! Koreshi far'en lai!" Simultaneously, the other dives at you, grabbing the gun out of your hands before you have a chance to fire it.`
+			`	Then, instead of attacking you, the one who took your gun begins waving it your face and talking very quickly, as if scolding you. It points to the sky with a sweeping gesture, then mimics drawing the gun as you just did. Then it points to the ground, puts the gun on the ground, and says, "Far'en lai ko forei lak'far." Then it picks up the gun and repeats both gestures: all the sky, gun drawn. Earth, gun on ground. Then it tosses the gun down at your feet and waits, expectantly.`
+				goto map
+
+			label map
+			`	There is a long awkward silence, like a cocktail party where you are standing next to someone but have no idea how to start a conversation. Then one of the Korath stoops down and begins drawing in the sand, a spiral with many arms: the galaxy. Near the center of the spiral, it draws the outline of one of their warships. "Korath soru nar," it says.`
+			choice
+				`	(Draw where human territory is.)`
+					goto draw
+				`	(Pretend not to understand what the map is.)`
+					goto wait
+
+			label draw
+			`	You draw the outline of your own ship on the same side of the core as theirs, but farther out, then draw a line around it marking the extents of human space. The Korath responds by drawing, beyond the southern edge of human space, something that looks like a spider, covering one whole galactic arm. Then, near the top of human space it pushes its finger deep into the ground, and again just north of there, and connects the two holes with a line.`
+			choice
+				`	(Add some details I know to the map.)`
+					goto add
+				`	(Wait to see what else they will draw.)`
+					goto wait
+
+			label add
+			# MZ wrote the parts about spiders, long before the Coalition was added to the game, the spiders are Archons. (so saith ravenshining)
+			`	Near the southern end of human space, you draw a dot with a broken ring around it: the Quarg ringworld, under construction. The Korath respond by both beginning to draw more rings, spread out all over the galaxy, some broken, some complete. Then they draw more of the spiders, again spread out all over the galaxy, and then sprinkle sand over them so the lines are barely visible.`
```

Comment:
> There is only one broken ring, the others which aren't complete are simply incomplete.
> Please rephrase this.

It's not unreasonable to say "broken ring" to describe an incomplete circle. This line goes on to say "under construction" already.

Though, "broken circle" may sound/read better since "ring" won't be repeated a few words later.

Reply:
Broken circle would be better, I just want to avoid having "broken ring" used as it currently means only one very specific thing when talking about Quarg ringworlds, an event fairly significant to the Quarg as a whole.

Reply:
I changed it to "partial ring" since "ring" is consistent with the rest of the mission (Quarg ringworld), and "partial" does not imply someone broke it. How is that?